translate/filters/checks.py

```diff
@@ -2155,10 +2161,31 @@ def accelerators(self, str1, str2):
         """Checks whether accelerators are consistent between the
         two strings.
 
-        For Mozilla we lower the severity to cosmetic.
+        For Mozilla we lower the severity to cosmetic, and for some languages
+        it also ensures accelerators are absent in the target string since some
+        languages do not use accelerators, for example Indic languages.
         """
+        if self.config.language_script:  # no-accelerators check
```

Comment:
Will adjust by adding a list of scripts the MozillaChecker skips in the accelerators check. Will keep this list though to figure out if the language uses a script that should be skipped.